Source/AlchemyDataNewsV1/Models/SAORelation.swift

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright IBM Corporation 2015
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import Freddy
+
+/**
+ 
+ **SAORelation**
+ 
+ Extracted Subject, Action, and Object parts of a sentence
+ 
+ */
+
+public struct SAORelation: JSONDecodable {
+    public let action: Action?
```

Comment:
We should comment these properties so that they don't appear as "Undocumented" in the Jazzy docs.